register-computer/join.ps1

```diff
@@ -44,7 +44,15 @@ $JoinInfoRedacted = $JoinInfo.PSObject.copy()
 $JoinInfoRedacted.ComputerPassword = "*"
 $JoinInfoRedacted | Format-List
 
-if ($JoinInfo.ComputerName.Length -gt 15) {
+$NewComputerName = $JoinInfo.ComputerName
+$OriginalComputerName = $JoinInfo.OriginalComputerName
+
+if ($NewComputerName -ne $OriginalComputerName) {
+    Write-Host "Renaming computer from $OriginalComputerName to $NewComputerName"
+    Rename-Computer -ComputerName localhost -NewName $NewComputerName -Force -PassThru -Verbose
```

Comment:
It's probably better to use `-NewName` in the `Add-Computer` call

Reply:
Tried to user -newname, but add-computer fails with "incorrect user name or password"